reV/econ/econ.py

```diff
@@ -241,46 +240,27 @@ def _econ_append_pc(pp, cf_file, sites_per_worker=None):
         pc : reV.config.project_points.PointsControl
             PointsControl object instance.
         """
-        try:
-            multi_h5_res, hsds = check_res_file(cf_file)
-            if multi_h5_res:
-                res_cls = MultiFileResource
-                res_kwargs = {}
-            else:
-                res_cls = Resource
-                res_kwargs = {'hsds': hsds}
-
-            with res_cls(cf_file, **res_kwargs) as f:
-                if 'gid' in f.meta:
-                    gid0 = f.meta['gid'].values[0]
-                    gid1 = f.meta['gid'].values[-1]
-                    if (gid0 in pp.df.gid.values
-                            and gid1 in pp.df.gid.values):
-                        # this is the scenario where the meta is a subset
-                        # of the project points such as when econ is being
-                        # run off a single node in append mode with the
-                        # full generation project points.
-                        i0 = pp.index(gid0)
-                        i1 = pp.index(gid1) + 1
-                        pc = PointsControl.split(
-                            i0, i1, pp, sites_per_split=sites_per_worker)
-                    else:
-                        # this is the scenario where the project points
-                        # is a subset of the meta data like for
-                        # test problems.
-                        pc = PointsControl(
-                            pp, sites_per_split=sites_per_worker)
-
-        except FileNotFoundError as ex:
-            msg = ('{} is invalid PointsControl can not be adjusted!\n{}'
-                   .format(cf_file, ex))
-            raise ProjectControlFileError(msg)
+        multi_h5_res, hsds = check_res_file(cf_file)
```

Comment:
This looks all good. Only suggestion would be to update the doc string to describe what an append-mode points control actually needs to be (just match the single-node generation output meta data).